main-site/components/events-calendar-section/EventsCalendarSection.styles.ts

```diff
@@ -2,9 +2,9 @@ import styled from "styled-components";
 import { max, min } from "../../../shared-ui/lib/responsive";
 import { colors } from "../../../shared-ui/style/colors";
 import { fonts, P } from '../../../shared-ui/style/typography';
-import { H3 } from '../../../shared-ui/style/typography';
 
 const StyledSectionContainer = styled.div`
+text-align: center;
```

Comment:
I think this centered ur entire section

It looked like this before
<img width="1014" alt="Screen Shot 2022-10-24 at 11 33 57 PM" src="https://user-images.githubusercontent.com/59738880/197676235-514baeea-4e2b-404b-be85-3d2acb20d04d.png">

Now it looks like this
<img width="990" alt="Screen Shot 2022-10-24 at 11 36 41 PM" src="https://user-images.githubusercontent.com/59738880/197676552-f97d1fa7-2bb2-4c40-8801-b3e5bc440aac.png">

Make sure you're only centering the section title here!